resources/views/deletefaq.blade.php

```diff
@@ -0,0 +1,25 @@
+<?php
+use Illuminate\Support\Facades\DB
+?>
+
+<x-layout>
+    <x-slot:title>
+        Delete FAQ Post {{ $post }}
+    </x-slot>
+    <form method="post" action="/deletefaqpost">
+        @method('put')
+        @csrf
+        <h1>Vraag: </h1>
+        <input type="text" name="question" value="{{ DB::table('faq')->where('id', $post)->value('question') }}">
+        <h1>Antwoord: </h1>
+        <input type="text" name="answer" value="{{ DB::table('faq')->where('id', $post)->value('answer') }}">
+        <input type="hidden" name="id" value="{{ $post }}">
```

Comment:
Somewhere in the theory we explain RESTful resource routes which means that it is not a good practice to put the id of the faq that must be showed, edited or deleted.